frontend/src/cljs/imas_seamap/map/utils.cljs

```diff
@@ -255,3 +231,30 @@
   "Sorts a collection by its sort-key first and its id second."
   [coll]
   (sort-by (juxt #(or (:sort_key %) "zzzzzzzzzz") :id) coll))
+
+(defn normal-latitude
+  "Latitude can get pretty wacky if one loops around the entire globe a few times.
+   This puts the latitude within the normal latitude range."
+  [lat]
+  (-> lat
+   (+ 180)
```

Comment:
Trivial, but this indentation feels off

Reply:
Whoops, yep. Trivial though. I have an idea of how it happened.